pkg/services/quota/quota.go

```diff
@@ -118,40 +128,107 @@ func (qs *QuotaService) getQuotaScopes(target string) ([]models.QuotaScope, erro
 	switch target {
 	case "user":
 		scopes = append(scopes,
-			models.QuotaScope{Name: "global", Target: target, DefaultLimit: qs.Cfg.Quota.Global.User},
-			models.QuotaScope{Name: "org", Target: "org_user", DefaultLimit: qs.Cfg.Quota.Org.User},
+			models.QuotaScope{
+				Name:             "global",
+				Target:           target,
+				TargetConditions: qs.getQuotaScopeConditions(target),
+				DefaultLimit:     qs.Cfg.Quota.Global.User,
+			},
+			models.QuotaScope{
+				Name:             "org",
+				Target:           "org_user",
+				TargetConditions: qs.getQuotaScopeConditions(target),
+				DefaultLimit:     qs.Cfg.Quota.Org.User,
+			},
```

Comment:
Think we can skip this for now since not needed. TargetConditions will be empty by default